modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -3,100 +3,208 @@
+Both paths rely on conda for installing the moose environment, therefore we recommend the
+users follow [Conda MOOSE Environment](installation/conda.md) instructions before starting
+this process.
 
-In situations when the `libc` version allows the linking but the compiler stack
-has been compiled with an older `libc` version (HPC machines potentially), we need to build the
-compiler from scratch. For instructions in this process, visit [installation/manual_installation_gcc.md]
+!alert! note title=GPU Support on Macs
 
-## Installing Libtorch
+For ARM Mac workstations, both installation procedures will give access
+to the Metal Performance Shader (MPS) capabilities (GPU acceleration).
 
-The user can choose from two alternatives when it comes to installing `libtorch`:
+!alert-end!
 
-- +Install using the script provided in MOOSE:+
+### Using packages distributed through conda
 
-  For this, navigate to the MOOSE root directory and execute the following script:
+1. Create and activate a new MOOSE environment with pytorch included:
 
-  ```bash
-  ./scripts/setup_libtorch.sh
-  ```
+   ```bash
+   conda create -n moose-torch moose-dev pytorch=2.1 -c pytorch
+   conda activate moose-torch
+   ```
 
-  which downloads `libtorch` from the official site and sets it up in the `framework\contrib`
-  directory. The script checks for operating system and `libc` version and throws errors
-  if the system is not suitable for the coupling.
+   This will provide the headers and libraries needed to use libtorch.
 
-  !alert! note
-  The desired version of libtorch can be set by the following argument:
+2. Configure MOOSE to use libtorch. The user needs to link to the conda-based libtorch libraries
+   using the following command within the root directory of `moose`:
 
-  ```bash
-  ./scripts/setup_libtorch.sh --version=1.8
-  ```
+   ```bash
+   ./configure --with-libtorch=${CONDA_PREFIX}/lib/python3.11/site-packages/torch
+   ```
 
-  Note that we do not support `libtorch` below a version of 1.4. The default
-  version downloaded by the script is 1.10.
+   !alert! note title=How to get the pytorch directory
 
-  !alert-end!
+   The python version can be different depending on the distribution, so make sure you double-check if the directory you point to actually exists!
+   An easy way to find if the library exists within the conda package is running the following command in the terminal:
 
-- +Install from source:+
-  The user can find instructions on how to install libtorch from source on the
-  [official website](https://github.com/pytorch/pytorch/blob/master/docs/libtorch.rst).
+   ```bash
+   find ${CONDA_PREFIX} -type d -name torch
+   ```
 
+   Alternatively, you can use python to get the same directory:
 
-## Configure and compile MOOSE with libtorch
+   ```bash
+   python -c "import torch; print(torch.__path__[0])"
+   ```
 
-To achieve this, first configure MOOSE with `libtorch` support (along with any other desired configure options)
+   !alert-end!
```

Comment:
I have two comments:
1. Is `python3.11` always the correct python version?
2. Can we further simplify this? Right now we are asking users to manually specify the libtorch path, but I think we have all the information needed to automatically figure it out. The main challenge is to detect whether the user is using a conda torch or a precompiled torch. But that can also be handled by either assuming a default or create different configure options like `--with-precompiled-libtorch` and `--with-conda-pytorch`.

Reply:
I am okay letting number 2 slide.

Reply:
For (1): Python 3.11 is the latest version we support, I'll remove this. It will change over time.

Reply:
For (2): We can definitely do that in the future. I would not like to add it to this PR, considering that it would also require changes to some of the testing infrastructure/containers. I'll open a new PR for that soon.

Reply:
> For (1): Python 3.11 is the latest version we support, I'll remove this. It will change over time.

I don't know if we could replace the hard-coded `3.11` with some substitution like ${latest_python_version}. I'm not sure if moosedocs supports that. I just wanted to prevent the instructions from becoming obsolete when we update python.

Reply:
Hmm I think we have that variable in moose docs and I didn't use it before because it had a rendering bug when it was used in a code block.

Reply:
Yeah, let's try not to increase our maintenance burden. A fallback plan is to let user figure this out with `python -V` (though that's less ideal apparently).

Reply:
Yeah to be honest you don't have to pin it at this point it seems to work on macs. We will have tests for this.